lib/challenge.js

```diff
@@ -227,13 +274,30 @@ let nums = [
   15,
   12,
 ];
+//I need to sort the numbers-- lets say there are odd numbers, then I need to figure out the middle number post sort-- and present that, if they are even i need to add the two middle numbers divide them and present that.
+//using multiple sources here-- geeksforgeeks, w3resources and fellow classmates to figure out sorting and capturing the median -- ive given up on this one.
+  function prompt11(){
+    // let median = Math.floor(newMid)
+    // let newMid = nums[(nums.length)/2] + nums[((nums.length)/2)+1]
+    // let median = nums;
+    // median = median.sort()
+    // console.log(median);
+    // var length = median.length;
+    // if(length % 2 == 1) {
+    //   console.log(median[(length / 2) - .5])
+    //   return median[(length / 2) - .5]
+    // }else{
+    //   console.log(median[length / 2] + median[(length / 2) - 1] / 2);
+    //   return(median[length / 2] + median[(length / 2) - 1]) / 2;
+    // }
+  }
+  // return nums.length % 2 !== 0 new[med] : (new[med - 1] + new[med])/2;
+  
```

Comment:
I get it, this one is really tricky!  I will be focusing on this one in particular during the review 

Great job on giving this one great effort. You were really close, your logic is sound.  Don't forget to sort the numbers too

on line 285 you used `var` the modern standard is `let` or `const`  There may be code out in the wild that still has `var` so it may come up from time to time.  If this happens just refactor for your purposes to `let` or `const`